src/pages/address/[address]/index.page.tsx

```diff
@@ -0,0 +1,16 @@
+import { AddressDashboard } from 'components/AddressDashboard'
```

Comment:
wdyt of `account/` as route, then we can use same route for ens one day too

Reply:
done